jsonschema-validation.xml

```diff
@@ -1015,15 +1015,17 @@
 
             <section title="contentSchema">
                 <t>
-                    If the instance is a string, and if "contentMediaType" is present, this
-                    property contains a schema which describes the structure of the string.
+                    If the instance is a string containing valid JSON, and if "contentMediaType" is
+                    present, this property is used to validate the string. If the string is not
+                    valid JSON, then the validation fails.
                 </t>
                 <t>
                     This keyword MAY be used with any media type that can be mapped into
                     JSON Schema's data model.
                 </t>
                 <t>
-                    The value of this property SHOULD be ignored if "contentMediaType" is not present.
+                    The value of this property MUST be a schema. It is ignored if "contentMediaType"
```

Comment:
You should (SHOULD 😛 ) keep the "SHOULD be ignored" wording.  

Another minor consistency nitpick:  Throughout the core spec, the phrasing is "MUST be a valid JSON Schema" (which shows you how far off I was when I wrote this section!)

Reply:
Done both.